criu/include/kerndat.h

```diff
@@ -69,6 +69,17 @@ struct kerndat_s {
 	bool has_clone3_set_tid;
 	bool has_timens;
 	bool has_newifindex;
+	/*
+	 * For non-root CRIU we read CLOCK_MONOTONIC and assume
+	 * that it can only move forward to detect a reboot.
+	 * Hopefully no one is using non-root CRIU in a timens.
+	 * Usually kdat cache is stored on tmpfs, for the non-root
+	 * use case we probably cannot access /run/. So we store it
+	 * ~/.criu.kdat and if the clock moves backward we assume
+	 * the system has been rebooted and the kdat cache needs
+	 * to be recreated.
+	 */
+	time_t clock_monotonic;
```

Comment:
clock_monotonic is started from 0 after reboot, isn't it?

Reply:
I don't understand how it can help us to detect the old kdat file...

Reply:
It doesn't help. This is just wrong. Any recommendations how to detect a reboot to reload the cache file?

Reply:
switched stale detection to release and nodename from `uname()`.